### PR TITLE
Traceview

### PR DESCRIPTION
**Pull request by tiffanyb**

### CODE REVIEW

File: angrmanagement/data/trace_statistics.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
+from PySide2.QtGui import QColor
```
Comment (ltfish) [inline angrmanagement/data/trace_statistics.py]: It goes after `import logging ... from collections import defaultdict`,

File: angrmanagement/data/trace_statistics.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+class TraceFunc:
+    def __init__(self, bbl_addr=None, func_name=None):
+        self.bbl_addr = bbl_addr
+        self.func_name = func_name
+
+
+class TraceStatistics:
+
+    BBL_FILL_COLOR = QColor(00, 0xf0, 0xf0, 15)
```
Comment (ltfish) [inline angrmanagement/data/trace_statistics.py]: Why `00`? I'm ok with `0` or `0x0`, but `00` looks really weird.

File: angrmanagement/data/trace_statistics.py [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    BBL_BORDER_COLOR = QColor(00, 0xf0, 0xf0)
+
+    def __init__(self, workspace, trace):
+        self.workspace = workspace
+        self.trace = trace
+        self.trace_func = list()
```
Comment (ltfish) [inline angrmanagement/data/trace_statistics.py]: `self.trace_func = [ ]`

File: angrmanagement/data/trace_statistics.py [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+        self._func_color = dict()
```
Comment (ltfish) [inline angrmanagement/data/trace_statistics.py]: `self._func_color = { }`

File: angrmanagement/data/trace_statistics.py [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+        self.count = None
+        self._mark_color = dict()
```
Comment (ltfish) [inline angrmanagement/data/trace_statistics.py]: Same here, ` = { }`.

File: angrmanagement/data/trace_statistics.py [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+    def get_bbl_from_position(self, position):
+        return self.trace_func[position].bbl_addr
+
+    def get_func_name_from_position(self, position):
+        return self.trace_func[position].func_name
+
+    def _statistics(self, trace):
+        """
+        trace: bbl address list
```
Comment (ltfish) [inline angrmanagement/data/trace_statistics.py]: `:param trace: Basic block address list.`

File: angrmanagement/ui/main_window.py [hunk 7]
```diff
@@ -90,6 +91,9 @@ def __init__(self, file_to_open=None, parent=None):
 
         if file_to_open is not None:
             self.load_file(file_to_open)
+            self._file_menu.action_by_key('load_trace').enable()
+
+        self._file_menu.action_by_key('load_trace').enable()
```
Comment (ltfish) [inline angrmanagement/ui/main_window.py]: Why do we enable the action multiple times?

File: angrmanagement/ui/views/disassembly_view.py [hunk 8]
```diff
@@ -30,6 +30,7 @@ def __init__(self, workspace, *args, **kwargs):
 
         self._linear_viewer = None  # type: QLinearViewer
         self._flow_graph = None  # type: QDisasmGraph
+        self._trace_viewer = None # type: QTraceViewer
```
Comment (ltfish) [inline angrmanagement/ui/views/disassembly_view.py]: Double spaces before `#`.

File: angrmanagement/ui/widgets/qinstruction.py [hunk 9]
```diff
@@ -8,13 +8,18 @@
 from .qoperand import QOperand
 from ...utils import should_display_string_label, get_string_for_display, get_comment_for_display
 
+import logging
+l = logging.getLogger(name=__name__)
+l.setLevel('INFO')
```
Comment (ltfish) [inline angrmanagement/ui/widgets/qinstruction.py]: Do not explicitly call `logger.setLevel()` in committed code. Also, `import logging` should go to the top of this file.

**Comment by ltfish** [comment]
Good job!

For future branches: Please add a proper prefix to your branch name, e.g., `feat/` (for a new feature) or `fix/` (for a new bug fix).

**Comment by tiffanyb** [comment]
This's been long time and I finally rebase it to latest master. @ltfish

**Comment by ltfish** [comment]
These commits have been merged to master as part of PR #122.

**Pull request closed without merging**